### PR TITLE
RavenDB-13215: If db being deleted from group, display it as deleted …

### PR DESCRIPTION
**Pull request by garayx**
…only.

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts [hunk 1]
```diff
@@ -206,8 +206,14 @@ class manageDatabaseGroup extends viewModelBase {
             this.nodes([]);
             this.clearNodesList(false);
         }
-        
-        this.updateNodes(incomingDbInfo.nodes());
+
+        if (incomingDbInfo.deletionInProgress().length > 0) {
+            let nodeToDisplay = incomingDbInfo.nodes().filter(x => incomingDbInfo.deletionInProgress().indexOf(x.tag()) < 0);
```
Comment (grisha-kotler) [inline src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts]: const

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts]: *nodesToDisplay

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts [hunk 2]
```diff
@@ -206,8 +206,14 @@ class manageDatabaseGroup extends viewModelBase {
             this.nodes([]);
             this.clearNodesList(false);
         }
-        
-        this.updateNodes(incomingDbInfo.nodes());
+
+        if (incomingDbInfo.deletionInProgress().length > 0) {
```
Comment (grisha-kotler) [inline src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts]: const deletionInProgress = incomingDbInfo.deletionInProgress();

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

**Review by ml054 — CHANGES_REQUESTED** [review]
we also have RavenDB-13089 which targets for 4.1, so the question is if we should PR this against 4.1 as well? Additionally I think this should be server side issue - not studio side - since currently you are filtering some state you get from server. Why can't we filter that on server side?

**Comment by garayx** [comment]
will open a new PR with server side fix

**Pull request closed without merging**